index.html

```diff
@@ -32,23 +32,23 @@
 
               <div class="contact-icons">
                 <button type="button" title="icon-twitter" class="btn-twitter">
-                  <i class="fa-brands fa-twitter fa-2x icon-twitter"></i>
+                  <img src="assests/images/twitter.svg" alt="twitter" />
                 </button>
                 <button
                   type="button"
                   title="icon-linkedin"
                   class="btn-linkedin"
                 >
-                  <i class="fa-brands fa-linkedin-in fa-2x icon-linkedin"></i>
+                  <img src="assests/images/Linkedin icon.svg" alt="Linkedin" />
                 </button>
                 <button type="button" title="icon-medium" class="btn-medium">
-                  <i class="fa-brands fa-medium fa-2x icon-medium"></i>
+                  <img src="assests/images/medium.svg" alt="medium" />
                 </button>
                 <button type="button" title="icon-github" class="btn-github">
-                  <i class="fa-brands fa-github fa-2x icon-github"></i>
+                  <img src="assests/images/github.svg" alt="github" />
                 </button>
                 <button type="button" title="icon-github" class="btn-angellist">
-                  <i class="fa-brands fa-angellist fa-2x icon-angellist"></i>
+                  <img src="assests/images/angellist.svg" alt="angellist" />
                 </button>
```

Comment:
- [ ] Nice attempt :+1: but I want to suggest that you use unordered list items -  `ul`/`li` tags for the structuring of your social media icons. This is because all are similar, and appear as a list of alternatives which for best practices, using a `ul`/`li` tag is better suited. It also forms one of the most important requirements for this implementation :ok_hand: